arrebol/worker/key/storage.go

```diff
@@ -0,0 +1,42 @@
+package key
```

Comment:
we have this file and the crypto one, but they should be only one, makes sense? And I think it shouldn't be in the worker directory, also it should be as more generic as possible

Reply:
I don't like this storage name as well..

Reply:
Crypto.go has generic functions for working with RSA keys. On the other hand, key/storage.go has specific functions to manipulate workers' keys.

Reply:
Didn't you like the file name or the interface name?

Reply:
I didn't like "storage"...

Reply:
Removed.